OpenSim/Examples/BuildDynamicWalker/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ cmake_minimum_required(VERSION 3.2)
 project(BuildDynamicWalker)
 
 # OpenSim requires a compiler that supports C++11.
```

Comment:
The associated comments (here, and below) should also be updated to reflect the update to C++14.